core/block_render_svg_vertical.js

```diff
@@ -857,6 +940,55 @@ Blockly.BlockSvg.prototype.renderDrawRight_ = function(steps,
 };
 
 /**
+ * Render the shape for an input.
+ * If there's a connected block, hide the input shape.
+ * Otherwise, draw and set the position of the input shape.
+ * @param {!Blockly.Input} input Input to be rendered.
+ * @param {Number} x X offset of input.
+ * @param {Number} y Y offset of input.
+ * @return {Number} Amount cursor should be offset (depending on width of render)
```

Comment:
Can you clarify this comment?  It should definitely say it's in the x direction, but I'm still not really sure what it's saying.